<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WuguanUIClass</class>
 <widget class="QWidget" name="WuguanUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="wuguanWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>584</height>
    </rect>
   </property>
   <widget class="QWidget" name="jichuzhengxingWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>220</width>
      <height>35</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="jichuzhengxingCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>基础整形</string>
     </property>
    </widget>
    <widget class="QLabel" name="jichuzhengxingNumberLabel">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>18</y>
       <width>25</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="jichuzhengxingSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>18</y>
       <width>190</width>
       <height>17</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="jichuzhengxingResetBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>2</y>
       <width>13</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/mainui/flush_button);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gaojizhengxingWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>63</y>
      <width>220</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QWidget" name="gaojiWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>220</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="gaojizhengxingCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>高级整形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaojizhengxingResetBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>42</y>
      <width>13</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/mainui/flush_button);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../joy.qrc"/>
 </resources>
 <connections/>
</ui>
